webapp/src/errorList.tsx

```diff
@@ -43,7 +43,7 @@ export class ErrorList extends React.Component<ErrorListProps, ErrorListState> {
         const createOnErrorMessageClick = (e: pxtc.KsDiagnostic, index: number) => () =>
             this.onErrorMessageClick(e, index)
 
-        const errorListHeader = <div className="errorListHeader">
+        const errorListHeader = <div className="errorListHeader" role="button" onClick={this.onCollapseClick}>
```

Comment:
add this parameter onKeyDown={sui.fire.... you’ll find it in many other UI elements.

Reply:
is this something that I should add to any element that is also a button?